lib/components/app/desktop-nav.tsx

```diff
@@ -68,16 +77,14 @@ const DesktopNav = ({ otpConfig, popupTarget, setPopupContent }: Props) => {
 
           {brandingOrTitle}
         </Navbar.Brand>
-      </Navbar.Header>
-      {/* @ts-expect-error typescript is having some trouble with complex type merges here */}
-      <ViewSwitcher sticky />
+        {/* @ts-expect-error typescript is having some trouble with complex type merges here */}
+        <ViewSwitcher sticky />
 
-      <Navbar.Collapse>
```

Comment:
Removing `Navbar.Collapse` is causing the nav bar to slightly expand vertically at widths less than 768px

![Screen Recording 2023-01-20 at 1 01 47 PM](https://user-images.githubusercontent.com/56846598/213773877-a2f71755-ac24-4c16-b15d-0b7f2851e842.gif)

Reply:
Yep I noticed this.... I chalked it up to something that's better worth fixing by replacing the entire navbar? Which we want to do anyway

Reply:
For the time being, add `display: none` at widths less than 768px to the underlying `<ul>` that has the top-right menus, please.

Reply:
Bug seems to have been introduced prior to this PR, so that can be fixed separately.